docs/source/torch/features/feature_combination_matrix.md

```diff
@@ -0,0 +1,18 @@
+| CUDA Graph                 | Yes               | ---        |                            |                       |                 |     |                           |                           |               |                  |                |                        |                       |           |
+| Attention Data Parallelism | Yes               | Yes        | ---                        |                       |                 |     |                           |                           |               |                  |                |                        |                       |           |
+| Disaggregated Serving      | Yes               | Yes        | Yes                        | ---                   |                 |     |                           |                           |               |                  |                |                        |                       |           |
+| Chunked Prefill            | No                | Yes        | Yes                        | Untested              | ---             |     |                           |                           |               |                  |                |                        |                       |           |
+| MTP                        | Yes               | Yes        | Yes                        | Yes                   | Untested        | --- |                           |                           |               |                  |                |                        |                       |           |
+| EAGLE-3(One Model Engine)  | Yes               | Yes        | Yes                        | No                    | Untested        | No  | ---                       |                           |               |                  |                |                        |                       |           |
+| EAGLE-3(Two Model Engine)  | NO                | Yes        | Yes                        | No                    | Untested        | No  | No                        | ---                       |               |                  |                |                        |                       |           |
+| Torch Sampler              | Yes               | Yes        | Yes                        | Yes                   | Yes             | Yes | Yes                       | Yes                       | ---           |                  |                |                        |                       |           |
+| TLLM C++ Sampler           | Yes               | Yes        | Yes                        | No                    | Yes             | No  | No                        | No                        | No            | ---              |                |                        |                       |           |
```

Comment:
@dcampora  Could you please help review the TLLM C++ Sampler part? To make sure the latest status is catched up.

Reply:
I would imagine that kv cache reuse is orthogonal to the choice of sampler, I am not aware of any code in the sampler where kv cache reuse is affected, so both Torch Sampler and TRTLLM Sampler should be supported. It would be better to have a test case of course.